projects/RabbitMQ.Client/Impl/RabbitMQActivitySource.cs

```diff
@@ -128,11 +131,11 @@ public static class RabbitMQActivitySource
 
             // Extract the PropagationContext of the upstream parent from the message headers.
             Activity? activity = s_subscriberSource.StartLinkedRabbitMQActivity(
-                UseRoutingKeyAsOperationName ? $"{routingKey} deliver" : "deliver",
+                UseRoutingKeyAsOperationName ? $"{routingKey} {MessagingOperationTypeProcess}" : MessagingOperationTypeProcess,
```

Comment:
this one is probably for `consume`, so let's do `consume|basic_consume|... {routing_key}` and add the operation name tag with the name we'll pick